.github/workflows/bindings.yml

```diff
@@ -52,19 +52,28 @@ jobs:
           cd crypto-ffi/bindings
           ./gradlew android:build -x lint -x lintRelease
 
-  check-swift:
+  build-swift:
     runs-on: macos-latest
+    strategy:
+      matrix:
+        task:
+          - name: ios-device
+            target: aarch64-apple-ios
+          - name: ios-simulator-x86
+            target: x86_64-apple-ios
+          - name: ios-simulator-arm
+            target: aarch64-apple-ios-sim
     steps:
       - uses: actions/checkout@v4
-      - uses: actions-rust-lang/setup-rust-toolchain@v1  # this implicitly caches Rust tools and build artifacts
+      - uses: actions-rust-lang/setup-rust-toolchain@v1
         with:
           rustflags: ''
-          target: "aarch64-apple-ios,x86_64-apple-ios,aarch64-apple-ios-sim"
+          target: ${{ matrix.task.target }}
       - name: Setup cargo-make
         uses: davidB/rust-cargo-make@v1
       - run: |
           cd crypto-ffi
```

Comment:
Maybe we can add a `- name` here too? I find it confusing that in the overview this is called `cd crypto-ffi`

Reply:
Totally. :+1:  :-)

Reply:
Done.